### DISCUSSION TITLE
Access Github Actions workflow durations

### DISCUSSION THREAD

**Discussion by dunyakirkali** [community/community#24781] [Apps, API and Webhooks]
<p>Hi :wave:</p>
<p>Is it possible to get stats on the Github Actions Workflows?</p>
<p>Specifically I’m trying to see how long are <code>Continuous Integration</code> and <code>Continuous Delivery</code> steps take (as in minutes and seconds). The same info which is visible in the UI?</p>

**Comment by alzafacon** [comment, marked as answer]
<p>Yes it is! Here’s a link to the documentation <a href="https://docs.github.com/en/rest/reference/actions#get-workflow-run-usage" class="inline-onebox" rel="noopener nofollow ugc">Actions - GitHub Docs</a></p>

**Comment by dunyakirkali** [comment]
<p>Awesome!</p>
<p>Thnx <a class="mention" href="/alzafacon">@alzafacon</a></p>
